src/pyodide/internal/python.ts

```diff
@@ -205,9 +220,12 @@ async function prepareWasmLinearMemory(Module) {
   adjustSysPath(Module);
 }
 
-export async function loadPyodide(lockfile, indexURL) {
+export async function loadPyodide(
+  lockfile: PackageLock,
+  indexURL: string,
+): Promise<Pyodide> {
   const emscriptenSettings = getEmscriptenSettings(lockfile, indexURL);
-  const Module = await enterJaegerSpan("instantiate_emscripten", () =>
+  const Module: Module = await enterJaegerSpan("instantiate_emscripten", () =>
```

Comment:
It shouldn't really be needed to annotate this variable since it will be inferred right?